src/js/components/Form/__tests__/Form-test-uncontrolled.js

```diff
@@ -277,8 +363,8 @@ describe('Form uncontrolled', () => {
     const onSubmit = jest.fn();
     const { getByText, queryByText } = render(
       <Grommet>
-        {/* this test continues running forever if the whole event 
-                passed to onSubmit */}
+        {/* this test continues running forever if the whole event passed to 
```

Comment:
Confused as to why this is here

Reply:
Can you update on the resolution?

Reply:
This comment was originally here and I changed the spacing which is why it is marked as being changed, to resolve it I just changed the spacing back to the original